pkg/kwokctl/utils/filepath_windows.go

```diff
@@ -0,0 +1,28 @@
+//go:build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+func PathJoin(elem ...string) string {
+	return strings.Replace(filepath.Join(elem...), `\`, `/`, -1)
```

Comment:
I'm confused on this: we want to modify the path because these are paths used within Docker/KinD running on windows correct?

What about user specifying `binary` when running on windows?

Reply:
> I'm confused on this: we want to modify the path because these are paths used within Docker/KinD running on windows correct?

Yes

> What about user specifying binary when running on windows?

Both `\` and `/` work on Windows.